api/package.json

```diff
@@ -12,7 +12,8 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "format": "prettier --config ../.prettierrc --ignore-path .prettierignore --write .",
-    "lint": "eslint src --ext .js,.ts"
+    "format:check": "prettier --config ../.prettierrc --ignore-path .prettierignore --check .",
```

Comment:
Doesn't lint already do this (i.e. any format warnings are already done in `npm run lint`)?